src/EnergyPlus/HeatBalFiniteDiffManager.cc

```diff
@@ -1470,6 +1497,8 @@ namespace HeatBalFiniteDiffManager {
 		if ( surface_ExtBoundCond == Ground || IsRain ) {
 			TDT( i ) = TT( i ) = TempOutsideAirFD( Surf );
 			RhoT( i ) = RhoVaporAirOut( Surf );
+			SurfaceFD( Surf ).CpDelXRhoS1( i ) = 0.0; // Outside face  does not have an outer half node
```

Comment:
Since it is just executing once per function, this lookup is likely not a big deal.  The best thing would be to probably take ```i``` out of the argument list and actually pass in instances of each thing you'll be looking up: ```TDT(i)```, ```TT(i)```, etc., and operate directly on all of those instances.  But that needs to happen across the board in this code and will not be a requirement for this enhancement.